apps/studio/src/lib/db/clients/sqlite.ts

```diff
@@ -69,6 +69,10 @@ export class SqliteClient extends BasicDatabaseClient<SqliteResult> {
     return this.version?.data[0]["sqlite_version()"];  
   }
 
+  defaultSchema(): string | null {
```

Comment:
Can this just be implemented like this in the `BasicDatabaseClient` instead? I think null is a sensible default.